### PR TITLE
Add edge-ami workflow jobs in RHEL-9.3 and CS9

### PR DESCRIPTION
**Pull request by mcattamoredhat**
This PR includes workflow jobs to trigger edge-ami CI tests in RHEL-9.3 and CS9.

### CODE REVIEW

File: .github/workflows/rhel-9-3.yml [hunk 1]
```diff
@@ -295,6 +296,90 @@ jobs:
...
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Runner has been deploying...'
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  edge-ami-image-93:
+    needs: [pr-info, pre-edge-ami-image-93]
+    runs-on: [kite, x86_64, rhos-01, rhel-9-3, large]
```
Comment (henrywang) [inline .github/workflows/rhel-9-3.yml:322]: Let's use medium one in this case.

**Review by henrywang — CHANGES_REQUESTED** [review]

File: .github/workflows/rhel-9-3.yml (hunk 1 above)
Comment (mcattamoredhat) [reply .github/workflows/rhel-9-3.yml]: Done!

**Comment by mcattamoredhat** [comment]
Failure in edge-simplified-installer-9.3 and edge-simplified-installer-cs9 is due to known FDO bug.

**Review by henrywang — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**